### DISCUSSION TITLE
YaroslavFedan / modules-inertia

### DISCUSSION THREAD

**Discussion by jobasco** [nWidart/laravel-modules#1451] [Ideas]
Hi, i am was searching for a way to use inertia+breeze+laravel-modules and came [https://github.com/YaroslavFedan/modules-inertia](url) but i am finding it difficult to understand/implement it so i want to ask if anyone has come across this modules-inertia or if someone is using it. Thanks.

**Comment by dcblogdev** [comment]
I don't use vue but it looks like you publish the stubs from the package then when generating modules the stubs should be the same as in the package ready to be used with vue/inertia

**Reply by jobasco** [reply]
Thanks, i have download the package but when i created a module it only gave me Resources/Pages/index.vue eventhough module/http/controllers/modulecontroller has the inertiarender, i have create a link to the module but the page is showing blank and also no errors.
